bootstrap/storageclass/helmfile/values/nfs-client-provisioner.yaml.gotmpl

```diff
@@ -9,4 +9,9 @@ serviceAccount:
   name: nfs-client-provisioner
 nfs:
   path: "/nfs"
-  #server: Will be set outside
+{{ if eq .Environment.Name "service_cluster" }}
+  server: {{ .Values.global.nfs_server_ip }}
+{{ end }}
+{{ if eq .Environment.Name "workload_cluster" }}
+  server: {{ .Values.global.nfs_server_ip }}
```

Comment:
Are the if-cases necessary? The value is `server: {{ .Values.global.nfs_server_ip }}` for both, right? :slightly_smiling_face: